webhooks/pod_webhook.go

```diff
@@ -304,166 +475,39 @@ func (m *PodMutator) enableClusterRoleBinding(ctx context.Context, pod *corev1.P
 	return nil
 }
 
-func (m *PodMutator) createConfigMap(ctx context.Context, name string, namespace string, pod *corev1.Pod) error {
+func (m *PodMutator) createConfigMap(ctx context.Context, namespace string, name string, pod *corev1.Pod) error {
 	m.Log.V(1).Info(fmt.Sprintf("Creating configmap %s", name))
 	references := []metav1.OwnerReference{
 		pod.OwnerReferences[0],
 	}
 	references[0].Controller = utils.FalseVal()
-	ff := m.getFeatureFlag(ctx, name, namespace)
-	if ff.Name != "" {
-		references = append(references, corev1alpha1.GetFfReference(&ff))
+	ff := m.getFeatureFlag(ctx, namespace, name)
+	if ff.Name == "" {
+		return fmt.Errorf("feature flag configuration %s/%s not found", namespace, name)
 	}
+	references = append(references, v1alpha1.GetFfReference(&ff))
 
-	cm := corev1alpha1.GenerateFfConfigMap(name, namespace, references, ff.Spec)
+	cm := v1alpha1.GenerateFfConfigMap(name, namespace, references, ff.Spec)
 
 	return m.Client.Create(ctx, &cm)
 }
 
-func (m *PodMutator) getFeatureFlag(ctx context.Context, name string, namespace string) corev1alpha1.FeatureFlagConfiguration {
-	ffConfig := corev1alpha1.FeatureFlagConfiguration{}
+func (m *PodMutator) getFeatureFlag(ctx context.Context, namespace string, name string) v1alpha1.FeatureFlagConfiguration {
```

Comment:
sanity check that the parameter flipping has been applied everywhere

Reply:
this is covered by the integration tests